src/stylesheets/Sponsor.scss

```diff
@@ -0,0 +1,37 @@
+.zoom:hover {
+    transform: scale(1.05); 
+}
+
+.imgContainer{
+    display:inline-block;
+    padding : 30px;
+}
+
+.center {
+    text-align: center;
+}
+
+.text {
+    padding : 15px;
+}
+
+.about-block--text {
+    text-align: justify;
+    max-width: 500px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .button_position {
+    text-align: center;
+    padding : 30px;
+  }
+
+  #gradient {
```

Comment:
Will have to change this to .gradient with modification outlined above

Reply:
Might actually be better to make a more specific className, since gradient is a bit generic